sys/linux/futex.txt.const

```diff
@@ -17,5 +19,6 @@ FUTEX_WAKE = 1
 FUTEX_WAKE_BITSET = 10
 FUTEX_WAKE_OP = 5
 __NR_futex = 98, 386:arm:240, amd64:202, mips64le:5194, ppc64le:221, s390x:238
+__NR_futex_waitv = 449, mips64le:5449, ppc64le:???
```

Comment:
what's `ppc64le:???`?

Reply:
futex_waitv hasn't been wired for PowerPC arch in the kernel until now. Should we do it?

Reply:
I wired it this morning! https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=b501b85957deb17f1fe0a861fee820255519d526